src/common-utils/main.sh

```diff
@@ -543,7 +542,11 @@ if [ "${INSTALL_ZSH}" = "true" ]; then
 
         # Add devcontainer .zshrc template
         if [ "$INSTALL_OH_MY_ZSH_CONFIG" = "true" ]; then
-            echo -e "$(cat "${template_path}")\nDISABLE_AUTO_UPDATE=true\nDISABLE_UPDATE_PROMPT=true" > ${user_rc_file}
+            if [ -f "${user_rc_file}" ]; then
+                echo -e "$(cat "${user_rc_file}")\n$(cat "${template_path}")\nDISABLE_AUTO_UPDATE=true\nDISABLE_UPDATE_PROMPT=true" > "${user_rc_file}"
+            else
+                echo -e "$(cat "${template_path}")\nDISABLE_AUTO_UPDATE=true\nDISABLE_UPDATE_PROMPT=true" > "${user_rc_file}"
+            fi
```

Comment:
There are many duplicates in this snippet, can we optimize the code?

Reply:
optimisation done. kindly review once.